module/VuFind/src/VuFind/Config/YamlReader.php

```diff
@@ -179,6 +182,22 @@ protected function parseYaml($file, $defaultParent = null)
             // Swallow the directive after processing it:
             unset($results['@parent_yaml']);
         }
+        // Override default parent with explicitly-defined parent, if present:
+        if (isset($results['@parent_config_name'])) {
+            $parentConfigName = $results['@parent_config_name'] . '.yaml';
+            $defaultParent = $useLocalConfig
```

Comment:
As currently implemented, if you specify both `@parent_yaml` and `@parent_config_name`, then `@parent_config_name` will override `@parent_yaml`. Should we document this order of precedence and/or issue a warning if both values are present (i.e. if $defaultParent is already set when we enter this second if statement)?